modules/sdk-coin-apt/src/lib/utils.ts

```diff
@@ -72,7 +72,7 @@ export class Utils implements BaseUtils {
       throw new Error('Invalid Payload: Expected TransactionPayloadEntryFunction');
     }
     const entryFunction = payload.entryFunction;
-    const moduleIdentifier = entryFunction.module_name.name.identifier.trim();
+    const moduleIdentifier = entryFunction.module_name.name.identifier;
     switch (moduleIdentifier) {
       case APTOS_ACCOUNT_MODULE:
```

Comment:
we have discussed to match the entire string `<module_name.address>::<module_name.name>::<function_name>`